src/System.Text.Json/tests/Serialization/Object.ReadTests.cs

```diff
@@ -337,12 +341,14 @@ public static void OuterClassHavingPropertiesDefinedAfterClassWithDictionaryTest
             Assert.Equal(2, parsedObject.MyInnerTestClass.MyInt);
         }
 
-        [ActiveIssue(38435)]
```

Comment:
So, this issue hasn't yet been fixed (https://github.com/dotnet/corefx/issues/38435). Why can we remove the `ActiveIssue` here now? Was this a different issue?

Reply:
It was same issue, i mean it was throwing same error `Collection was of a fixed size` but now it is not throwing that anymore, seems that dictionary related bug Tanner fixed was causing different Exception depending on json structure